src/Features/Core/Portable/Features.csproj

```diff
@@ -110,5 +110,8 @@
   <ItemGroup>
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="$(SystemThreadingTasksExtensionsVersion)" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="ConvertAnonymousTypeToTuple\" />
```

Comment:
is this needed?

Reply:
i made the folder for the common impl, but haven't populated  it yet.  so this is waht VS does.